### DISCUSSION TITLE
1756-L85E throws error T01:C61 Major Fault when communicating CIP

### DISCUSSION THREAD

**Discussion by WimVriend** [digitalpetri/ethernet-ip#60] [General]
Hello Kevin a.o.,

For a number of weeks now we have been programming the 1756-L85E processor (SER B  Firmware 33.011) and communicating with it using the CIP protocol. At first we were doing both debugging RsLogix and communicating with the SCADA simultaneously, using the onboard ethernet port. That did not go very well: every now and then the CPU would throw the T01:C61 error and it completely wiped the memory of the PLC in the process.

To isolate the problem, we then switched all communication to the 1756-EN2T module (SER D Revision 11.001). There we experienced no problems.

Today we made an attempt to investigate the matter some more. When debugging via the onboard port or the USB port with the SCADA on the 1756-EN2T, no problems occur. As soon as we move the SCADA communication back to the onboard port, the crashes occur again :(.

Is this a 'known issue'? Or have we stumbled upon on strange new bug? I assume that it is a problem in the CPU, since all works fine when using the 1756-EN2T?

**Comment by kevinherron** [comment]
Sounds like it could be a strange new firmware bug.

I haven't heard any reports of this, however it's unusual for controller faults to end up in our support inbox, as that's pretty much always a programming or firmware issue.

If I do hear about something similar I will come back to this discussion with an update.

**Reply by WimVriend** [reply]
Do you have any experience reporting such error to AllenBradley? I have read some stuff on the internet stating that they are not very grateful when you report them...

Since the communication and the PLC-program work fine when using the  1756-EN2T, I think we can rule out programming issues. So that leaves the firmware. If I were a computer company, I would be glad if users would report issues like these :)

**Reply by kevinherron** [reply]
No, I don’t have any experience trying to report a bug to them. I suspect it would be difficult, especially because my company is not a paying customer other than having a Tech Connect contract, and in many ways we are competitors to parts of their software business.
